src/cockpit/superuser.py

```diff
@@ -38,24 +36,24 @@
 class SuperuserStartup:
     peer: Peer  # the peer that's being started
 
-    def success(self, rule: SuperuserRoutingRule) -> None:
+    def success(self, rule: 'SuperuserRoutingRule') -> None:
```

Comment:
I see lots of conflicts in my near future....

Reply:
Yeah, that's why I'm aiming for putting this into CI. [almost there](https://github.com/cockpit-project/cockpit/actions/runs/4116286201/jobs/7106134717) :grin: